### PR TITLE
Update LIT1898Martyr.xml

### PR DESCRIPTION
**Pull request by DariaElagina**
According to the text, she was his mother, እሙ፡
Maybe I miss something...

### CODE REVIEW

File: 1001-2000/LIT1898Martyr.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1898Martyr">
 
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Martyrium of Mamas, Tewodoṭos and his wife Tewofina</title>
+            <title xml:id="t1">Martyrium of Mamas, Tewodoṭos and his mother Tewofina</title>
```
Comment (eu-genia) [inline 1001-2000/LIT1898Martyr.xml:8]: Tewodotos/Theodotus was Mamas' father, so Tewofina (Rufina in other traditions) was the latter's wife, so it was correct

If we are at correcting the record - do we translate the names or do we transcribe the names? (I guess either, in English, Mamas/Mammes, Theodotus, etc, or, transcribed, then it should be Māmās, or then two titles)

https://en.wikipedia.org/wiki/Mammes_of_Caesarea

maybe at this point we can add wd:Q532019 and life dates (259?-275?)

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
Mamas, Tēwodoṭos, Tēwofina – 5 Maskaram
GS (13 mss.)
BL Or. 686 = BLW 257: 2 (1755-69): ff. 5rc-10vc (f. 5r: 5 Maskaram);
BL Or. 687-688 = BLW 258: 2 (XVIII s.): ff. 9ra-15r (f. 9r: 5 Maskaram);
BL Or. 689 = BLW 253: 2 (XV s.): ff. 7vb-15ra; (f. 7v: 5 Maskaram);
BNZ 131: 1 (XIII s.): ff. 1ra-18va (f. 1r: 5 Maskaram); BNZ ("ወኮነ፡ ስምዖሙ፡ ለኵሎሙ፡ ቅዱሳን፡ በሕማም፡ ወምንዳቤ፡ ወከመ፡ አባግዕ፡ ዘይጠብኁ፡ ከማሁ፡ ይገብሩ፡ ላዕለ፡ ሰማዕቱ፡ ለክርስቶስ። ወበፍሥሓ፡ ወሐሤት፡ ፍጡነ፡ ይረውጹ፡ ውስተ፡ ስማዕ። / ወኮነ፡ በውእቱ፡ መዋዕል፡ ብእሲ፡ እምሰብአ፡ ሀገር፡ እንተ፡ ስማ፡ ቴቅላ፡ ዘመንገለ፡ ምዕራበ፡ ብሔረ፡ ሮሜ። ወዝንቱሰ፡ ቴዎዶጦስ፡ ወብእሲቱ፡ ቴዎፊና፡ እምዐበይት፡ ወእምቀደምት፡ እለ፡ የኀድሩ፡ ውስተ፡ ይእቲ፡ ሀገር፡ ወጥቀ፡ ሠናይ፡ ግብሮሙ፡ … […] Comparez Surius, «Vitae Sanctorum», 17 aug., t. III, 173 […] Les feuillets ont été transposés lors de la reliure. Ils doivent ētre placés dans l’ordre suivant 1, 13, 15, 11-12, 16, 14, 2-10, 17-18");
BNA 110: 3 = BNAC 110: 2 = BNACR 164: 2 (XVIII s.): ff. 8ra-14vc (5 Maskaram);
EMML 1479: 2 (1459-60): ff. 7vb-15va;
EMML 1766: 2 (XIV-XV s.): ff. 14ra-27va;
EMML 2514: 10 (1382-88): ff. 44ra-50va; EMML (በስመ፡ አብ፡ … ዝንቱ፡ ዜናሁ፡ ለቅዱስ፡ ማማስ፡ ሰማዕት፡ ለኵሎሙ፡ ቅዱሳን፡ በሕማም፡ ወምንዳቤ፡ ወከመ፡ አባግዕ፡ The day of their commemoration is not given in the upper margin of the folio where the texts starts. According to the Synaxary, they are commemorated on 5 Maskaram; cf. also Wright (B.M.) no. CCLVII, 2, p. 166. Here, too, the text is copied immediately after that of the Beheading of John the Baptist, 2 Maskaram; the manuscripts has been incorrectly rebound. It is not clear, therefore, what the date 24 Ṭeqemt, 345 ‘Amata Meḥrat, with which the text end (f. 50b), stands for);
EMML 6903: 2 (XV s.): ff. 11ra-20v (f. 11r: 5 Maskaram);
EMML 6951: 1 (XV s.): ff. 2ra-8vb (f. 2r: 5 Maskaram);
EMML 6965: 2 (XIV s.): ff. 8ra-15vb (f. 8r: 5 Maskaram); "በ፫፻፡ ፵ወ፭፡ ዓመተ፡ ምሕረት፡ እምተወ፬፡ ለወርኀ፡ ጥቅምት።። nel 345esimo anno di grazia, dal 24 del mese di Ṭeqemt";
Ṭānāsee 121 = Dāgā Esṭifānos 10: 2 (1416): A: I: 2: ff. 10ra-19ra (f. 10r: 5 Maskaram);
Dabra Libānos (XV s.) 2;
Ṭānāsee 105 = Rēmā 16 ṬānāH Vol.II: 151 (framm.) […] "I. Bl. 1r-v, 4r-v und 272r-v (hier als Deckblät-ter verwandt) sind ein Fragment, das wahrscheinlich zu der Hs. Ṭānāsee 92 (II) = o. 121f. gehört: ein Teil der Akten des ማማስ፡ [< Μάμας] mit seinem Vater ቴዎዶጦስ፡ [< Θεόδοτος] und seiner Mutter ቴዎፊና፡ [< arab.: twfynā < rwfynā < Ῥουφίνη], Märtyrer unter Kaiser Aurelian. Ihre Akten sind ein Bestandteil des Sammelwerkes ገድለ፡ ሰማዕታት። [= Viten der Märtyrer] […]".

Arabo
GCAL I.520.

File: 1001-2000/LIT1898Martyr.xml (hunk 1 above)
Comment (DariaElagina) [reply 1001-2000/LIT1898Martyr.xml]: I think that the English title is a little bit intransparent, "his" can refer to both persons...

**Comment by DariaElagina** [comment]
https://github.com/BetaMasaheft/Documentation/issues/1856

**Review by eu-genia — APPROVED** [review]

**Pull request merged**